corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -412,7 +421,12 @@ FormplayerFrontend.on('refreshApplication', function(appId) {
         tfLoadingComplete(true);
     }).done(function() {
         tfLoadingComplete();
-        FormplayerFrontend.trigger('navigateHome');
+        if (typeof resp.responseJSON.tree !== 'undefined') {
```

Comment:
this is kind of an awkward check. can just use: http://underscorejs.org/#isUndefined